tequilapi/http_api_server.go

```diff
@@ -64,6 +65,14 @@ func (server *apiServer) Wait() error {
 	return <-server.errorChannel
 }
 
+// Port method returns bind port for given http server (useful when random port is used)
+func (server *apiServer) Address() (string, error) {
```

Comment:
Maybe Port() and Address() functions can be combined to single one which can return both bound port, address or error?